src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-manager.route.ts

```diff
@@ -99,5 +99,10 @@ export const GERMPLASM_MANAGER_ROUTES: Routes = [
         path: 'germplasm-import',
         component: GermplasmImportPopupComponent,
         outlet: 'popup'
+    },
+    {
+        path: 'lot-creation',
+        component: LotCreationPopupComponent,
+        outlet: 'popup',
     }
 ];
```

Comment:
You can also place it below `lot-creation-dialog` route, for cohesion

Reply:
Done